python/cross_service/apigateway_websocket_chat/test/test_lambda_chat.py

```diff
@@ -87,7 +87,7 @@ def test_handle_message(
     )
     if error_method != "stub_scan":
         apig_management_stubber.stub_post_to_connection(
-            f"{user_name}: {msg}".encode("utf-8"),
```

Comment:
Same as above for encoding.

Reply:
Based on my [research](https://peps.python.org/pep-0529/), this would only cause an issue if someone is using Windows pre-10:
```
Python 3.6 is already using UTF-8 by default on macOS, Android and Windows
```
Which OS are you concerned about?

Reply:
Same answer as above.

Reply:
Added comment.